src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -98,6 +100,43 @@ export default function FormNav(props) {
     : `Step ${currentChapterDisplay} of ${chaptersLengthDisplay}: ${chapterName ||
         ''}`;
 
+  const handleFocus = async () => {
+    let root = document.querySelector('#react-root');
+    if (formConfig.v3SegmentedProgressBar) {
+      // Need to provide shadowRoot for focusing on shadow-DOM elements
+      const shadowHost = await querySelectorWithShadowRoot(
+        'va-segmented-progress-bar',
+      );
+      root = shadowHost.shadowRoot;
+    }
+
+    return () => {
```

Comment:
Focus wasn't working for me on the review page (tested mock-simple-forms-patterns). I can help test some more pages tomorrow

Reply:
Hmm, this seems to be returning a function from `handleFocus`, so the useEffect call on line 137 doesn't do anything. Then it seems that [this line of code is causing focus to move to the `h2`](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L32) in the footer of our page (the "Need help?" header). Maybe we need to remove this, because once this `handleFocus` function works, we don't need to manage focus in the `RoutedSavableReviewPage` component.

Reply:
Refactors made, but couldn't get the appeals forms to work on my local.  Pinged Robin to test.